src/Documenter.jl

```diff
@@ -325,14 +329,23 @@ function deploydocs(;
 
         repo   = error("no 'repo' keyword provided."),
         branch = "gh-pages",
-        latest = "master",
+        latest::Union{String,Nothing} = nothing, # deprecated
 
         osname = "linux",
         julia  = error("no 'julia' keyword provided."),
 
         deps   = Deps.pip("pygments", "mkdocs"),
         make   = () -> run(`mkdocs build`),
+
+        devbranch = "master",
+        devurl = "dev",
     )
+    # deprecation of latest kwarg (renamed to devbranch)
+    if latest !== nothing
+        Base.depwarn("The `latest` keyword argument have been renamed to `devbranch`.")
+        devbranch = latest
```

Comment:
One silly edge case: if the user defines both `latest` and `devbranch` simultaneously. Might be good to error here just in case.

Reply:
I don't think we can know this, if the user have set `devbranch` to the defaul. I'll just add an `@info` here saying which branch we choose.